composer.json

```diff
@@ -24,12 +24,13 @@
     "league/commonmark": "^2.0",
     "scrivo/highlight.php": "^v9.18",
     "paquettg/php-html-parser": "^3.1",
-    "php": "8.0.*",
+    "php": "8.1.*",
     "illuminate/support": "^8.69"
   },
   "require-dev": {
     "alleyinteractive/alley-coding-standards": "^0.3.0",
-    "yoast/phpunit-polyfills": "^2.0"
+    "phpunit/phpunit": "^9.6",
+    "yoast/phpunit-polyfills": "^1.0"
```

Comment:
Why is the phpunit-polyfills downgraded?

Reply:
The Polyfills 2.0 package throws warnings for PHP features not supported in PHPUnit 10, even if we're testing with PHPUnit 9.